src/pages/RFOX/types/index.ts

```diff
@@ -1,3 +1,10 @@
 export type AddressSelectionValues = {
   manualRuneAddress: string | undefined
 }
+
+export enum StepIndex {
```

Comment:
Probably revert me unless we use the tab indexes in a way this declarativeness may be useful, but doesn't look like we will in the end